.github/workflows/_build-img-nucliadb.yml

```diff
@@ -9,6 +9,11 @@ on:
         type: string
         required: false
         default: ""
+      image-version:
```

Comment:
Is this parameter used anywhere? I cannot see it.

Reply:
it shold be used, but I was calculating the SHORT_SHA twice, one from the deploy workflow and again on this flow. I have done a change, let's rely on the one that is pass from the deploy workflow.